docs/reference/self-hosted.md

```diff
@@ -13,8 +16,24 @@ A remote Bit Scope is a host server that hosts components. Bit communication to
 
 ## Server Setup
 
+<Tabs
+defaultValue="Harmony"
+values={[
+{label: 'Harmony', value: 'Harmony'},
+{label: 'Legacy', value: 'Legacy'},
```

Comment:
yeah we should remove this. not necessary

Reply:
done